### ISSUE TITLE
It does not jump to the target heading in mobile

### ISSUE THREAD

**Issue by findfirecode** [cqroot/joplin-outline#98]
Not working in mobile

**Comment by TheOtherGuy** [comment]
I do not get the option to add this plugin on the Joplin iOS app. Is it available on Android?

**Comment by Danie10** [comment]
Yes I got it in Android, and it shows perfectly well, as well as the highlight for where you touch it, but it does not jump to that location like on desktop. I hope it can be fixed as I have a few very long reference notes where the Outline really helps.
